demo/nodejs/file.js

```diff
@@ -19,20 +19,27 @@ const WaveFile = require("wavefile").WaveFile;
 const { Cheetah, getInt16Frames, checkWaveFile } = require("@picovoice/cheetah-node");
 
 program
-  .requiredOption(
+  .option(
     "-a, --access_key <string>",
     "AccessKey obtain from the Picovoice Console (https://console.picovoice.ai/)"
   )
-  .requiredOption(
+  .option(
     "-i, --input_audio_file_path <string>",
     "input wav file"
   )
+  .option(
+    "-y, --device <string>",
+    "Device to run inference on (`best`, `cpu:{num_threads}`, `gpu:{gpu_index}`). Default: selects best device for `pvcheetah`")
```

Comment:
`pvcheetah` is the name of the python package
```suggestion
    "Device to run inference on (`best`, `cpu:{num_threads}`, `gpu:{gpu_index}`). Default: selects best inference device")
```